pages/index.tsx

```diff
@@ -1,8 +1,180 @@
+import { InfoCircleFilled } from "@ant-design/icons";
+import { Select, Tooltip } from "antd";
+import { useRouter } from "next/router";
+import { FormEvent, useState } from "react";
+
 const Home = () => {
+  const [jobRole, setJobRole] = useState("");
+  const [skills, setSkills] = useState([]);
+  const [yearsOfExperience, setYearsOfExperience] = useState(0);
+  const [jobDescription, setJobDescription] = useState("");
+  const router = useRouter();
+
+  const onSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    router.push({
+      pathname: "/interview",
+      query: { skills, yearsOfExperience, jobDescription, jobRole },
+    });
+  };
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="z-10 w-full max-w-5xl items-center justify-between font-mono text-lg lg:flex">
-        <p>TechTalQ</p>
+    <main className="h-screen flex flex-col items-center justify-between p-24">
+      <div className="z-10 w-1/2 max-w-5xl font-mono">
+        <h1 className="mb-5 text-2xl font-bold text-center text-primary-300">
+          TechTalQ
+        </h1>
+
+        <form className="mt-8 space-y-6" onSubmit={onSubmit}>
+          <div className="rounded-md shadow-sm">
+            <div className="mb-4">
+              <label htmlFor="skills">
+                Job Role*
+                <Tooltip title="'Job role' refers to the specific position or title that an interview candidate is applying for within the organization.">
+                  <InfoCircleFilled className="ml-2 align-top" />
+                </Tooltip>
+              </label>
+              <Select
+                id="jobRole"
+                className="appearance-none rounded relative
+                  w-full
+                  placeholder-gray-500 text-gray-900
+                  focus:outline-none focus:ring-primary-300 bg-primary-300
+                  focus:border-primary-300x focus:z-10 sm:text-sm"
+                placeholder="Please select"
+                value={jobRole}
+                onChange={(values) => setJobRole(values)}
+                options={[
+                  { label: "Software Engineer", value: "Software Engineer" },
```

Comment:
Move this and skills in a JSON file and import from there.
`
{

> "Roles": ["Software Developer", "Backend Engineer"...]
"Skills": ["C#", "NodeJs" ......]
}
`

Also get a random 20 Job roles and about 100 technologies to start with and put in the Json File

Reply:
done